lib/utils.js

```diff
@@ -244,10 +245,7 @@ var caseInsensitiveEquals = function (str1, str2) {
     return str1.toUpperCase() === str2.toUpperCase();
 };
 
-var serializeXhrBody = function (xhrInfo, body) {
-
-    // TODO implement better FormData serialization for websocket
-    // see: https://github.com/form-data/form-data
+var serializeXhrBody = function (headers, body, seed) {
```

Comment:
What is the use of `seed` I don't see it is being passed from any method in this PR?